gbs-control.ino

```diff
@@ -20,6 +20,30 @@
 #include "framesync.h"
 #include "osd.h"
 
+#incldue "SSD1306Wire.h"
+#include "fonts.h"
+#include "images.h"
+SSD1306Wire display(0x3c,D2,D1);
+const int pin_clk = 14; //D5 GPIO14
+const int pin_data = 13; //D7 GPIO13
+const int pin_switch = 0; //D3 GPIO0 pulled HIGH, else fail on boot
+
+int menuItem = 1;
+int subsetFrame = 0;
+int selectOption = 0;
+int page = 0;
+
+String menu[4] = {"Resolutions","Presets","Misc.","Current Settings"};
+String Resolutions[7] = {"1280x960","1280x1024","1280x720","1920x1080","480/576","Downscale","Pass-Through"};
+String Presets[8] = {"1","2","3","4","5","6","7","Back"};
+String Misc[4] = {"Reset GBS","Restore Factory","-----Back"};
+
+int lastCount = 0;
+volatile int encoder_pos = 0;
```

Comment:
Can these OLED related variables get an "OLED" attachment? This is to clearly separate them from GBSC base code.

Reply:
Just simply adding on an OLED tag to them? Sure! For example, "volatile int oled_encoder_pos = 0"?

Reply:
yep, sounds good